### PR TITLE
bump vcpkg

### PR DESCRIPTION
**Pull request by qrkourier**

### CODE REVIEW

File: .github/workflows/docker-publish.yml [hunk 1]
```diff
@@ -73,6 +73,40 @@ jobs:
             # keep major version, e.g., v1, tag updated to latest release
             type=semver,pattern=v{{major}},enable=${{startsWith(github.ref, 'refs/tags/v')}}
 
+      - name: Find latest vcpkg version
```
Comment (scareything) [inline .github/workflows/docker-publish.yml:76]: I tried for a while to look for a way to get Dependabot to do this for us, but came up empty. I still think the Dependabot model where your get a dedicated PR for your dependency updates is the way to go.

Could we do these that you've added here in a new workflow that triggers on a schedule and uses something like [create-pr-action](https://github.com/technote-space/create-pr-action) to put up a PR with just the .vcpkg_version file updated?

Comment (qrkourier) [reply .github/workflows/docker-publish.yml:76]: I can add a scheduled workflow that calls this one and creates the PR if there's a new release.

**Comment by qrkourier** [comment]
@scareything this is ready for review

**Comment by qrkourier** [comment]
@scareything this is ready for review and introduces the scheduled vcpkg version checker you asked for

**Comment by qrkourier** [comment]
The version bumper workflow respects GH Actions var `VCPKG_VERSION`, which overrides the latest vcpkg release version.

**Review by scareything — APPROVED** [review]
nice!

**Pull request merged**